test/powershell/Scripting/NativeExecution/NativeCommandArguments.Tests.ps1

```diff
@@ -0,0 +1,31 @@
+Describe "Native Command Arguments" -tags "CI" {
+    $testPath = 'TestDrive:\test.txt'
+    $powershellTestDir = $PSScriptRoot
+    while ($powershellTestDir -notmatch 'test[\\/]powershell$') {
+        $powershellTestDir = Split-Path $powershellTestDir
+    }
+    $echoArgsDir = Join-Path (Split-Path $powershellTestDir) EchoArgs
+
+    function Start-EchoArgs {
+        Push-Location $echoArgsDir
+        try {
+            dotnet run $args
```

Comment:
To minimize interference with `dotnet` cli, it's desirable to replace `dotnet run` by `dotnet publish --output <path-to-EchoArgs>/run` and direct native executable call.
And we probably should move this step out from the tests to the `build.psm1`.

This way, tests would not depend directly on`dotnet` and it's presents in PATH